### PR TITLE
refactor: Change `OpIdentifier` scope to table instead of connection

### PR DESCRIPTION
**Pull request by chubei**

### CODE REVIEW

File: dozer-ingestion/tests/test_suite/basic.rs [hunk 1]
```diff
@@ -50,11 +54,18 @@ pub async fn run_test_suite_basic_data_ready<T: DataReadyConnectorTest>() {
...
-        if let IngestionMessageKind::OperationEvent { table_index, op } = &message.kind {
-            if let Some(last_identifier) = last_identifier {
-                assert!(message.identifier > last_identifier);
+        if let IngestionMessage::OperationEvent {
+            table_index,
+            op,
+            id,
+        } = &message
+        {
+            if let Some(id) = id {
```
Comment (Jesse-Bakker) [inline dozer-ingestion/tests/test_suite/basic.rs:63]:
```rust
if let Some((id, last_id)) = id.zip(last_identifier) /* or Option::zip(*id, last_identifier) */ {
    assert!(id > last_id)
}
```

File: dozer-ingestion/tests/test_suite/basic.rs [hunk 2]
```diff
@@ -158,17 +173,22 @@ pub async fn run_test_suite_basic_insert_only<T: InsertOnlyConnectorTest>() {
...
-            let IngestionMessageKind::OperationEvent { op: operation, .. } = message.kind else {
+            let IngestionMessage::OperationEvent {
+                op: operation, id, ..
+            } = message
+            else {
                 continue;
             };
 
+            // Identifier must be increasing.
+            if let Some(identifier) = last_identifier {
```
Comment (Jesse-Bakker) [inline dozer-ingestion/tests/test_suite/basic.rs:185]: Same as above

File: dozer-ingestion/tests/test_suite/basic.rs [hunk 3]
```diff
@@ -237,17 +257,22 @@ pub async fn run_test_suite_basic_cud<T: CudConnectorTest>() {
...
-        let IngestionMessageKind::OperationEvent { op: operation, .. } = message.kind else {
+        let IngestionMessage::OperationEvent {
+            op: operation, id, ..
+        } = message
+        else {
             continue;
         };
 
+        // Identifier must be increasing.
+        if let Some(identifier) = last_identifier {
```
Comment (Jesse-Bakker) [inline dozer-ingestion/tests/test_suite/basic.rs:269]: Same as above

File: dozer-core/src/checkpoint/mod.rs [hunk 4]
```diff
@@ -93,7 +116,7 @@ impl OptionCheckpoint {
     ) -> Result<Option<Vec<u8>>, storage::Error> {
         if let Some(checkpoint) = &self.checkpoint {
             let key = processor_key(&checkpoint.processor_prefix, node_handle);
-            info!("Restoring processor {node_handle} from {key}");
+            info!("Downloading processor {node_handle} checkpoint from {key}");
```
[the remark this answers is not in the document]
Comment (chubei) [reply dozer-core/src/checkpoint/mod.rs:119]: "Loading" is better. What's wrong about `.map(Some)`?

[the remark this answers is not in the document]
Comment (chubei) [reply dozer-core/src/checkpoint/mod.rs:119]: Haha, let's keep it there then? Saves quite some characters.

[the remark this answers is not in the document]
Comment (Jesse-Bakker) [reply dozer-core/src/checkpoint/mod.rs:119]: Sure

**Review by Jesse-Bakker — APPROVED** [review]

**Pull request merged**